### ISSUE TITLE
Svelte support

### ISSUE THREAD

**Issue by Dikower** [amcharts/amcharts4#2062]
Will you support Svelte in the future? I really love Amcharts for its design and the ease of use, but I had a lot of troubles with importing it into Svelte like manual rollup configuration and then cyclic dependency, so that makes impossible to use this library in my apps. But Chart.js works fine it needs only to be installed by yarn/npm

**Comment by martynasma** [comment]
At this point we do not have any plans for Svelte support.

**Comment by iambudi** [comment]
@Dikower do you have alternative charts compatible with Svelte?

**Comment by milahu** [comment]
TLDR - here is a repl - [amcharts4 in svelte demo](https://svelte.dev/repl/0b524d32b41247099699b8ab2344c6a2)

> @Dikower do you have alternative charts compatible with Svelte?

not really .. https://madewithsvelte.com/charts

fusioncharts is heavyweight / slow / hangs / makes my browser crash .... ancient stuff!
[svelte-fusioncharts.svelte](https://github.com/fusioncharts/svelte-fusioncharts/blob/develop/src/index.svelte)

frappe and pancake are rather underfeatured (no zoom/scroll, for example)
[svelte-frappe-charts.svelte](https://github.com/himynameisdave/svelte-frappe-charts/blob/master/src/components/base.svelte)

pancake is nice, cos its completely written in svelte
[pancake.svelte](https://github.com/Rich-Harris/pancake/blob/master/components/Chart.svelte)

here is a demo with chart.js 
https://codesandbox.io/s/svelte-chartjs-1eomo?file=/App.svelte
chartjs at least has a plugin for zoom/scroll

> manual rollup configuration

my rollup failed on dynamic imports .. in theory, rollup supports dynamic imports
but svelte can also be bundled by webpack https://github.com/sveltejs/template-webpack
